extensions/mssql/src/contracts.ts

```diff
@@ -1614,4 +1614,12 @@ export namespace RenameObjectRequest {
 	export const type = new RequestType<RenameObjectRequestParams, void, void, void>('objectManagement/rename');
 }
 
+export interface DropObjectRequestParams {
+	connectionUri: string;
+	objectUrn: string;
+}
+
+export namespace DropObjectRequest {
+	export const type = new RequestType<DropObjectRequestParams, void, void, void>('objectManagement/drop');
```

Comment:
STS is currently returning a bool, so you should either use that here as well or just remove it if you don't actually need it

https://github.com/microsoft/sqltoolsservice/pull/1953/files#diff-54c0d874540b352211a3ea778f6215db1b42f266d29b2db7d6d4fcc1e68d6179R28

Reply:
unfortunately in CS, we can't use void as generic type, I'll probably use an empty class in ADS side in a future PR.